README.md

```diff
@@ -302,6 +310,22 @@ This will run the [mypy][] static type checker, and then [pytest][].
 To edit the tests for `librecval`, see `tests/`. For Django tests, look
 inside the `validation/tests` directory.
 
+### Cypress Tests
+
+There are also Cypress integrations tests that can be run by doing:
+
+```shell
+npx cypress open
+```
+This opens the interactive Cypress testing window. Select `Run integration specs` to run 
+all tests.
+
+> **Note:** These tests will run against whichever database is present. Meaning if you're developing
+> locally, they will use that database; if you're running on production, they will use that database. **DO NOT** run
+> Cypress tests on Production: this will validate some entries due to the nature of the tests.
+> 
+> Additionally, at least 1 test will always fail: either one of the `register` tests or one of the `login` 
+> tests. This is because a) the username is already taken or b) the user does not exist. This is fine.
```

Comment:
Is there a way to:
 1. Have cypress verify that it is *not* running in production
 2. Use a randomly-generated username so that the tests pass normally?

Reply:
Ideally, I would have Cypress tests running on a test database so I could guarantee which users exist and what permissions they have, but I don't know how to do that and it's not near the top of my list of priorities.

Reply:
Ok, I understand. My main concern was that the wording suggested the possibility of running against production before saying not to; how about shortening it to the warning:

```suggestion
> **Note: DO NOT** try to run Cypress tests on production: this will validate
> some entries due to the nature of the tests.
```

Reply:
Perfect, I'll change it so it's clear.